### ISSUE TITLE
Feature request: Jeep Engine Remote Start

### ISSUE THREAD

**Issue by mattdevo1** [wubbl0rz/FiatChamp#81]
Any chance you could implement engine remote start/stop for Jeep gas vehicles? @wubbl0rz?

This is an option in the official Jeep app (see screenshot below), so it is presumably supported by the API. 

I'm willing to help test and even pay for your time to do it. This would make the integration perfect for me (everything else works great!)

Thank you. 

![image](https://github.com/user-attachments/assets/f3e9ad8a-425c-4fc0-8247-b33711eb6871)

**Comment by nficarella** [comment]
@mattdevo1 You got the integration to show your Jeep in MQTT?

**Comment by mattdevo1** [comment]
Yes. Honestly I set it up about a year ago, and don't exactly remember how I did it. But I have sensors for things like tire pressure, and door lock/unlock works. But it doesn't seem to expose engine start/stop (which you can do in the Jeep app).

It looks like this integration/add-on is kind of dead though. Would love for someone to pick this up and continue development. I'm happy to help test, but I don't really have the skills to maintain.

**Comment by nficarella** [comment]
I understand, i just tried recently and it will not populate into my MQTT broker

I also agree i would pay to get this functionally working again with HA along with starting and stopping the car

**Comment by smarthomefamilyverrips** [comment]
@wubbl0rz would you be willing to pick up this integration up again?

**Comment by Blueion76** [comment]
@nficarella @mattdevo1 @smarthomefamilyverrips I have implemented jeep engine remote start/stop in my forked repo: https://github.com/Blueion76/FCAUconnect-HA

**Comment by mattdevo1** [comment]
> @nficarella @mattdevo1 @smarthomefamilyverrips I have implemented jeep engine remote start/stop in my forked repo: https://github.com/Blueion76/FCAUconnect-HA

Awesome!! I'll try it on the weekend.

Any other changes we should be aware of in your fork?

Also (this is a big one...), this integration seems to be dead at this point. Any thoughts on taking it over?

**Comment by Blueion76** [comment]
> > @nficarella @mattdevo1 @smarthomefamilyverrips I have implemented jeep engine remote start/stop in my forked repo: https://github.com/Blueion76/FCAUconnect-HA
> 
> Awesome!! I'll try it on the weekend.
> 
> Any other changes we should be aware of in your fork?
> 
> Also (this is a big one...), this integration seems to be dead at this point. Any thoughts on taking it over?

There shouldn't be any major changes. I did change a couple things related to MQTT but I believe I reverted that, can't remember as it was a couple of weeks ago haha. I'd be happy to take over the integration if others submit pull requests to fix issues or if it's something simple. I'm not an expert though so some things may be out of the realm of possibility without others help. :)
